### PR TITLE
user/vivid: new package

### PR DESCRIPTION
**Pull request by jabuxas**

### CODE REVIEW

File: contrib/vivid/template.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+pkgdesc = "Themeable LS_COLORS generator"
+maintainer = "jabuxas <jabuxas@proton.me>"
+license = "MIT AND Apache-2.0"
+url = "https://github.com/sharkdp/vivid"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "88db6158dad60aba66ae16f2cd1b09f515625940a33bada65da5562a03538e49"
+
+
+def post_install(self):
+    self.install_license("LICENSE-APACHE")
```
Comment (nekopsykose) [inline contrib/vivid/template.py:16]:
```suggestion
```

Comment (jabuxas) [reply contrib/vivid/template.py:16]: uhmm why? main/rust installs both licenses

Comment (nekopsykose) [reply contrib/vivid/template.py:16]: because apache doesn't have to be installed

Comment (jabuxas) [reply contrib/vivid/template.py:16]: ok so why does rust do it

Comment (nekopsykose) [reply contrib/vivid/template.py:16]: because someone put it there when they didn't have to

Comment (jabuxas) [reply contrib/vivid/template.py:16]: sure

**Pull request merged**